### PR TITLE
[Docs] Batch08: adding examples to string functions

### PR DESCRIPTION
**Pull request by edgar2020**
### Description

Add examples to the sql-functions.md page

Updates the `REPLACE`, `REPEAT`, `STRPOS`, `REVERSE`, `STRING_FORMAT`, `SUBSTRING`, `SUBSTR`, `TRIM`, `BTRIM`, `LTRIM` and `RTRIM` functions.

This PR has:

- [X] been self-reviewed.

### CODE REVIEW

File: docs/querying/sql-functions.md [hunk 1]
```diff
@@ -1519,27 +1655,69 @@ Alias for [`LENGTH`](#length).
 
 ## STRPOS
 
-`STRPOS(<CHARACTER>, <CHARACTER>)`
+Returns the one-based index position of a substring within an expression.
```
Comment (FrankChen021) [inline docs/querying/sql-functions.md:1658]: what if the substring is not found in the given expression?

File: docs/querying/sql-functions.md [hunk 2]
```diff
@@ -1706,17 +1884,36 @@ Takes the difference between two timestamps, returning the results in the given
 **Function type:** [JSON](sql-json-functions.md)
 
 `TO_JSON_STRING(expr)`
-
+ƒ()
```
Comment (FrankChen021) [inline docs/querying/sql-functions.md:1887]: what is this?

Comment (edgar2020) [reply docs/querying/sql-functions.md:1887]: thank you for catching this!

**Review by FrankChen021 — APPROVED** [review]

**Comment by FrankChen021** [comment]
@edgar2020 could you resolve conflicts?

**Comment by edgar2020** [comment]
> @edgar2020 could you resolve conflicts?

@FrankChen021 Merge conflicts resolved

**Pull request merged**